app/Team.php

```diff
@@ -8,6 +8,7 @@
 class Team extends Model
 {
 
+    const MINIMUM_PLAYERS = 4;
```

Comment:
Should this be configurable or is it a strict requirement?

Reply:
It could. I didn't think it would belong in the .env file. I could make a config file for it. My think was it changes so infrequently that it was easier to add as a constant.

Reply:
Nah, this sounds fine for now.